conda-recipe/meta.yaml

```diff
@@ -32,7 +31,6 @@ requirements:
     - numexpr  # required for pandas DataFrame.query, used in examples
```

Comment:
The conda recipe can be dropped altogether, it's an old version, the new one is maintained at conda-forge